doc/container.md

```diff
@@ -332,7 +332,7 @@ have to set manually:
     admin@example:/config/interface/docker0/> set container-network type bridge
 
 > **Note:** when doing the same operation over NETCONF there is no
-> inference, so all the "magic" settings needs to be defined.  This
+> inference, so all the "magic" settings need to be defined.  This
 > makes the CLI very useful for first setup and then extracting the
 > resulting XML from the shell using the `cfg -X` command.
 
```

Comment:
For future reference, Emacs Alt-Q (Meta-Q) is really good in Markdown mode to "reflow" paragraphs, even quote blocks like this :nerd_face: